doc/source/releases/24.0_announce.rst

```diff
@@ -1,12 +1,61 @@
 
-:orphan:
+===========================================================
+24.0 Galaxy Release (March 2024)
+===========================================================
+
+.. include:: _header.rst
+
+Highlights
+===========================================================
+
+Feature1
+--------
+
+Feature description.
+
+Feature2
+--------
+
+Feature description.
+
+Feature3
+--------
+
+Feature description.
+
+Are you an admin? See the Administration Notes below, and check out `some admin relevant PRs <https://github.com/galaxyproject/galaxy/pulls?q=label%3Ahighlight%2Fadmin+milestone%3A24.0+is%3Aclosed+is%3Apr>`__.
 
+Get Galaxy
 ===========================================================
-24.0 Galaxy Release (Early 2024)
+
+The code lives at `GitHub <https://github.com/galaxyproject/galaxy>`__ and you should have `Git <https://git-scm.com/>`__ to obtain it.
+
+To get a new Galaxy repository run:
+  .. code-block:: shell
+
+      $ git clone -b release_24.0 https://github.com/galaxyproject/galaxy.git
+
+To update an existing Galaxy repository run:
+  .. code-block:: shell
+
+      $ git fetch origin && git checkout release_24.0 && git pull --ff-only origin release_24.0
+
+See the `community hub <https://galaxyproject.org/develop/source-code/>`__ for additional details on source code locations.
+
+Administration Notes
+===========================================================
+
+[TODO: add items or drop section if none]
```

Comment:
Not sure if this is the place, but should we add a comment here for https://github.com/galaxyproject/galaxy/pull/16662 to make sure admins remember to add this new tool to their existing tool_conf.xml?

Reply:
Yes, I think so. Last time [we had a bullet list](https://docs.galaxyproject.org/en/master/releases/23.2_announce.html#administration-notes) of admin-related changes (most of those came from PRs labeled as "highlight/admin")

Reply:
Ok, I added the "highlight/admin" label to the PR